src/main/java/de/samply/bbmri/negotiator/control/owner/OwnerQueriesDetailBean.java

```diff
@@ -274,12 +275,12 @@ public String initialize() {
 			// Get the selected(clicked on) query from the list of queries for the owner
 			// Needs to be optimized!
 
-			for(OwnerQueryStatsDTO ownerQueryStatsDTO : getQueries()) {
-				if(ownerQueryStatsDTO.getQuery().getId() == queryId) {
-					selectedQuery = ownerQueryStatsDTO.getQuery();
-					setCommentCountAndUreadCommentCount(ownerQueryStatsDTO);
-				}
-			}
+//			for(OwnerQueryStatsDTO ownerQueryStatsDTO : getQueries()) {
```

Comment:
please delete it if it's no longer necessary

Reply:
Yes thangs deleted the commented code